svelte/lib/src/components/slider/SliderDefaultStructure.svelte

```diff
@@ -33,11 +33,11 @@
 		use:widget.directives.minLabelDirective
```

Comment:
This section is also interesting: someone creating a slider with headless would probably have a different DOM, does it make sense to create directives for all those sub-elements ?